galsim_extra/mixed_scene.py

```diff
@@ -58,8 +59,23 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
         # object ends up being chosen.
         ignore = ignore + ['objects', 'magnify', 'shear', 'obj_type']
 
+        stamp_xsize, stamp_ysize, image_pos, world_pos = super(MixedSceneBuilder, self).setup(config,base,xsize,ysize,ignore,logger)
+        # option to shear the full scene.
+        if config.get('shear_full_scene', True):            
+            shear = galsim.Shear(g1=float(base["stamp"]["shear"]["g1"]), g2=float(base["stamp"]["shear"]["g2"]))
+            S = shear.getMatrix()
+            # Find the center (tangent point) of the scene in RA, DEC. 
+            scene_center = base['wcs'].center
```

Comment:
This isn't a thing.  Or, it is a thing for GSFitsWCS, but it's an arbitrary place on the sky that doesn't have any relevance to us scientifically.  It's just an implementation detail of the particular WCS solution.

You probably want to use the center of the image.  That's `base['world_center']`